nativeauthenticator/handlers.py

```diff
@@ -266,12 +273,24 @@ async def get(self):
     @web.authenticated
     async def post(self):
         user = await self.get_current_user()
-        new_password = self.get_body_argument("password", strip=False)
-        success = self.authenticator.change_password(user.name, new_password)
+        old_password = self.get_body_argument("old_password", strip=False)
+        new_password = self.get_body_argument("new_password", strip=False)
+        confirmation = self.get_body_argument("con_password", strip=False)
```

Comment:
The new version reads as follows, I hope that is okay. 

```python
        old_password = self.get_body_argument("old_password", strip=False)
        new_password = self.get_body_argument("new_password", strip=False)
        confirmation = self.get_body_argument("new_password_confirmation", strip=False)
```

I kept the variable name at `confirmation` instead of also going for `new_password_confirmation` because it is pleasantly the same length as both `old_password` and `new_password`, creating a nice visual indicator that these three are basically the same. I do not believe this to be confusing.